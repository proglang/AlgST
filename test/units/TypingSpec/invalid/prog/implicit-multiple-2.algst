calc : ({Int} -> Int) -> Int

{a} : Int
{a} = 10

synth : Int
synth =
  let r = calc (implicit[Int]) in
  r

check : Int
check = calc (implicit[Int])
