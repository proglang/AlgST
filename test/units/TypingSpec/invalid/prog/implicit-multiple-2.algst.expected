
8:26: error:
	 Multiple chocies for implicit of type Int
	   • «implicit-arg»
	     bound at 8:26
	   • a
	     imported from M

12:9: error:
	 Multiple chocies for implicit of type Int
	   • «implicit-arg»
	     bound at 12:9
	   • a
	     imported from M